src/main/java/com/sourcegraph/website/SearchActionBase.java

```diff
@@ -31,10 +32,14 @@ public void actionPerformedMode(@NotNull AnActionEvent event, @NotNull Scope sco
     if (selectedText == null || selectedText.length() == 0) {
       return;
     }
+    Editor editor =
+        Arrays.stream(CodyEditorUtil.getSelectedEditors(project)).findFirst().orElse(null);
+    if (editor == null) {
+      return;
+    }
```

Comment:
Is there no `firstOrNull()`? instead of `findFirst().orElse(null)`?

Can't we early return in the expression context with `?:`

```suggestion
    Editor editor =
        Arrays.stream(CodyEditorUtil.getSelectedEditors(project)).findFirst().orElse(null) ?: return
```